src/components/FolderBar.js

```diff
@@ -1,10 +1,32 @@
+import { useEffect, useState } from "react";
+import { getFolder } from "../api";
 import "./FolderBar.css";
 
-export default function FolderBar({ folderName, userName, imgSrc }) {
+export default function FolderBar() {
+  const [folderName, setFolderName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [profileImage, setProfileImage] = useState("");
+
+  useEffect(() => {
+    async function getProFileFolder() {
+      try {
+        const {
+          folder: { name, owner },
+        } = await getFolder();
+        setFolderName(name);
+        setUserName(owner.name);
+        setProfileImage(owner.profileImageSource);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+
+    getProFileFolder();
+  }, []);
   return (
     <div className="FolderBar">
       <div className="user">
-        <img id="folderImg" src={imgSrc} alt="폴더 이미지"></img>
+        <img id="folderImg" src={profileImage} alt="폴더 이미지"></img>
```

Comment:
eslint 설정을 하면 되지만, 코드를 작성할때도 요런 포인트들은 신경써주시면 좋을 것 같아요~
```suggestion
        <img id="folderImg" src={profileImage} alt="폴더 이미지" />
```